quickwit/quickwit-metastore/src/split_metadata.rs

```diff
@@ -40,8 +40,9 @@ pub struct Split {
     /// Timestamp for tracking when the split was published.
     pub publish_timestamp: Option<i64>,
 
-    /// Immutable part of the split.
     #[serde(flatten)]
+    #[schema(inline, value_type = VersionedSplitMetadata)]
```

Comment:
Why did you use `inline` here?

Reply:
The schema is flattened by serde, so we use inline here to reflect what serde is doing.

Reply:
utoipa supports serde flatten (see [PR](https://github.com/juhaku/utoipa/pull/325/files), and I'm not sure if inline is doing what you think it does. When putting inline, it will put the whole schema definition in the object and will not refer to a component schema.